apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -121,6 +121,28 @@ export function InstructorCourseAdminInstances({
                 </tbody>
               </table>
             </div>
+            ${courseInstances.length === 0
+              ? html`
+                  <div class="my-4">
```

Comment:
At specific viewports, the text comes really close to the side of the panel:

<img width="676" alt="Screenshot 2024-12-10 at 09 16 25" src="https://github.com/user-attachments/assets/019421ca-4aed-4458-8325-deb76499610f">

I'd recommend using the `card-body` class here to provide appropriate padding: https://getbootstrap.com/docs/5.3/components/card/#body

Reply:
I added the card-body padding:

<img width="405" alt="image" src="https://github.com/user-attachments/assets/dcd0c28c-c730-4be4-bba7-e9788586874c">